File: Estrutura_Sequencial/C++/Ex1007/main.cpp
#include <iostream>

int main(){

    int A, B, C, D, Diferenca;
    std::cin >> A;
    std::cin >> B;
    std::cin >> C;
    std::cin >> D;
    Diferenca = A * B - C * D;

    std::cout << "DIFERENCA = " << Diferenca << std::endl;
    
    return 0;
}

File: Estrutura_Sequencial/Java/Ex1007/Main.java
package Ex1007;

import java.util.Scanner;

public class Main {

    public void main() {
        Scanner sc = new Scanner(System.in);
        int A = sc.nextInt();
        int B = sc.nextInt();
        int C = sc.nextInt();
        int D = sc.nextInt();
        sc.close();
        int diferenca = A * B - C * D;

        System.out.printf("DIFERENCA = %d\n", diferenca);
    }

    public static void main(String[] args) {
        Main m = new Main();
        m.main();
    }
}